package.json

```
@@ -14,8 +14,31 @@
     "node": ">=11.14.0"
   },
   "categories": [
-    "Other"
+    "Other",
+    "Debuggers"
   ],
+  "contributes": {
+    "configuration": [
+      {
+        "type": "object",
+        "properties": {
+          "codeart-binexplore.objdumpPath": {
+            "type": "string",
+            "default": "/usr/bin/objdump",
```

It looks like `spawn` uses $PATH
```suggestion
            "default": "objdump",
```

---

Done